nautobot/extras/tests/test_api.py

```diff
@@ -2466,6 +2472,30 @@ def setUpTestData(cls):
                 source_type=site_type,
                 destination_type=device_type,
             ),
+            Relationship(
+                name="VLANs require at least one device (destination objects)",
+                slug="vlans-devices",
+                type="many-to-many",
+                source_type=vlan_type,
+                destination_type=device_type,
+                required_side="destination",
+            ),
```

Comment:
I find this feature really confusing, not sure if I'm misunderstanding it, or if this is wrong logic - the description of `DESTINATION_SIDE_REQUIRED` is `Destination objects MUST implement this relationship`, which I would interpret in this case as meaning that each Device in this relationship must specify at least one VLAN - but that's the opposite of what you're naming this relationship, above. Is the name wrong, is the help text wrong, or am I misunderstanding this feature?

Reply:
I think I confused myself by inverting things as you commented here: https://github.com/nautobot/nautobot/pull/2371#discussion_r981244943